pkg/registry/registry.go

```diff
@@ -30,6 +30,13 @@ const (
 	IANAReversedEnterpriseID uint32 = 29305
 )
 
+// enum for flowEndReason field in IANA registry.
+const (
+	IdleTimeoutReason   = uint8(0x01)
+	ActiveTimeoutReason = uint8(0x02)
+	EndOfFlowReason     = uint8(0x03)
+)
```

Comment:
We can support all the reasons here in case of future need.
https://www.iana.org/assignments/ipfix/ipfix.xhtml#ipfix-flow-end-reason

Reply:
What is the default value for flowEndReason? If it is (0x00) we can have a const here too to be used in Antrea.

Reply:
It is a reserved value. Where do you think we can use that in Antrea? For all flow records, the above three reasons are sufficient.

Reply:
I'm thinking that for flows that are not terminated, what value should be assigned?

Reply:
For flows, which are not terminated we keep on sending the records with either activeFlow or idleFlow tags.
Are you seeing any more cases through your work in tcp state PR?

Reply:
Got it. I haven't seen other cases. Just want to make sure.